docs/reference/tracebox.md

```diff
@@ -4,97 +4,392 @@
+
+### Limitations
+
+- SDK-instrumented apps won't connect (uses private, PID-based sockets)
+- No persistent service for multiple traces
+- Less efficient for repeated tracing
+- Daemons terminate when tracing completes
+
+### When to Use
 
-`tracebox` can be used to invoke the bundled applets.
+- Quick one-off traces without daemon setup
+- Environments where persistent daemons aren't feasible
+- Testing or development scenarios
 
-The general syntax for using `tracebox` in *manual mode* is as follows:
+### Platform Support
 
+- **Linux/Android:** Abstract domain sockets (`@traced-c-PID`, `@traced-p-PID`)
+- **macOS:** Filesystem sockets (`/tmp/traced-c-PID`, `/tmp/traced-p-PID`)
+- **Windows:** Not supported
+
+### Example
+
+```bash
+tracebox --autodaemonize -t 10s -o trace.pftrace sched freq
 ```
- tracebox [applet_name] [args ...]
+
+## APPLET MODE
+
+Invoke bundled applets directly:
+
+```bash
+tracebox [applet_name] [args ...]
 ```
 
-The following applets are available:
+Available applets:
 
 `traced`
-:    The Perfetto tracing service daemon.
+:    The Perfetto tracing service daemon. See [traced(1)](traced.md).
 
 `traced_probes`
-:    Probes for system-wide tracing (ftrace, /proc pollers).
+:    System-wide tracing probes (ftrace, /proc pollers). See [traced_probes(1)](traced_probes.md).
 
 `traced_perf`
-:    Perf-based CPU profiling data source.
+:    Perf-based CPU profiling data source (Linux only).
 
 `perfetto`
-:    The commandline client for controlling tracing sessions.
+:    Command-line client for tracing sessions. See [perfetto(1)](perfetto-cli.md).
 
 `trigger_perfetto`
-:    A utility to activate triggers for a tracing session.
+:    Utility to activate triggers for tracing sessions.
 
 `websocket_bridge`
-:    A bridge for connecting to the tracing service via websockets.
+:    Bridge for connecting to tracing service via WebSockets.
+
+`ctl`
+:    Daemon lifecycle management (start/stop/status).
+
+### Examples
+
+```bash
+tracebox traced --help
+tracebox traced_probes --reset-ftrace
+tracebox perfetto -t 10s -o trace.pftrace sched
+```
+
+## PLATFORM-SPECIFIC BEHAVIOR
+
+### Android
+
+**Default:** Requires daemons to be running.
+- System daemons typically use `/dev/socket/traced_{producer,consumer}`
+- `tracebox ctl start` creates user-session daemons in `/tmp/`
+- Or use `--autodaemonize` for self-contained mode
+
+**Important:** On Android devices with system-wide Perfetto installed, daemons are
+usually already running at `/dev/socket/`. Use `tracebox ctl status` to check.
+
+**Note:** SDK-instrumented apps require system daemons and cannot connect to
+user-session daemons started via `tracebox ctl start`.
+
+### Linux/macOS
+
+**Default:** Requires daemons to be running.
+- Use `tracebox ctl start` to start persistent daemons
+- Or use `--autodaemonize` for self-contained mode
+
+### Windows
+
+**Default:** Always spawns daemons with system sockets.
```

Comment:
Why are we doing different things on Windows? We should just be consistent no? At least to the extent of saying on Windows --autodaemonize must be specified.

It sucks for everyone if you have inconsistent behaviour like this.